third_party/intel/backend/driver.py

```
@@ -6,21 +6,30 @@
 from triton.runtime.cache import get_cache_manager
 from triton.backends.compiler import GPUTarget
 from triton.backends.driver import DriverBase
+import pkg_resources
 
 ze_root = os.getenv("ZE_PATH", default="/usr/local")
 include_dir = [os.path.join(ze_root, "include")]
+library_dir = []
 
 oneapi_root = os.getenv("ONEAPI_ROOT")
 if oneapi_root:
     include_dir += [
         os.path.join(oneapi_root, "compiler/latest/include"),
         os.path.join(oneapi_root, "compiler/latest/include/sycl")
     ]
+else:
```

Shall we raise the exception if sycl runtime libraries and headers file folder location can't be identified at all?

---

I guess it is bad idea for the scenarios when `icpx` compiler is presented. We are adding `-fsycl` option that does all this magic for us.

---

However I've added `try-catch` for the case when there is no runtime package.

---

As discuss, it will introduce logic duplication.

---

Implemented as discuses: raise exception if sycl library is not found.